.github/workflows/int-test-workflow.yml

```diff
@@ -4,7 +4,46 @@ on:
     branches:
       - develop
       - master
+      - fix-splunk-image-pull
 jobs:
+  build-operator-image:
+    runs-on: ubuntu-latest
+    env:
+      SPLUNK_ENTERPRISE_IMAGE: ${{ secrets.SPLUNK_ENTERPRISE_IMAGE }}
+      SPLUNK_OPERATOR_IMAGE_NAME: splunk/splunk-operator
+      ECR_REPOSITORY: ${{ secrets.ECR_REPOSITORY }}
+      S3_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Would it be more clear or relevant if a "name: Checkout code" line is added above this one? Same question for the nightly-in-test file

Reply:
It really has not effect. It still shows up like this in workflow run.
![image](https://user-images.githubusercontent.com/69360493/175665027-87245803-8745-4700-a819-5bb83c683423.png)